code/modules/client/preference_setup/loadout/items/suit.dm

```diff
@@ -291,6 +291,8 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["dominian cape, military"]	= /obj/item/clothing/accessory/poncho/dominia_cape/mantle
```

Comment:
from the item's desc

> Commonly worn by enlisted soldiers of the Imperial military
> Many armsmen carry on this trend in order to maintain their Imperial Army roots

Reply:
What is it you're asking for here, sorry?

Reply:
this was just to highlight that the same point applies to it as to about the NCO patch

but I think this cape should be fine-ish, cause it's not explicitly like a ranked officer's cape or something, but just a general "this is what armsmen sometimes wear" cape (at least that's how I understand it)